fmtph264.go

```diff
@@ -0,0 +1,109 @@
+package webrtc
```

Comment:
`fmtp_h264.go` a little easier to read maybe?

Reply:
actually would it be possible to put this in `internal`?

Reply:
I'll look at putting it in internal. I'm not crazy about this solution in general, but I think a better solution would come with more codec support. For example, replacing the MediaEngine.RegisterCodec() arguments with an Interface where the Interface implements the various parsers and other Codec specific details. That way codec support can be external to pion/webrtc and codec implementations can live on their own. That is probably a v4 change though :)